src/__tests__/logFormatting.js

```diff
@@ -90,4 +95,10 @@ describe('calculateUserStatus', () => {
         expect(params.reStringWithNoMatch).toEqual(undefined);
         expect(params.sub2number).toEqual(undefined);
     });
+
+    it('should handle malformed requests', () => {
```

Comment:
Adding a little test coverage. I'll expand this in a follow up for better request parsing.